config/visionfive2-release-traps.toml

```diff
@@ -0,0 +1,25 @@
+# A simple configuration to run on StarFive VisionFive 2 platform in release mode
```

Comment:
We already have multiple VisionFive 2 configs, we need to clarify what this one does and why it is different from the others. It is not obvious from just reading the file for new people coming to the project.